### ISSUE TITLE
Patch Android2Po to understand ixl and tsz

### ISSUE THREAD

**Issue by pocmo** [mozilla-mobile/focus-android#2560]
From the import:

>    [failed] ixl is not a valid locale
>    [failed] tsz is not a valid locale

We have patched a2po before:
https://github.com/mozilla-mobile/focus-android/blob/master/tools/l10n/android2po/env.py#L58

**Comment by boek** [comment]
Hey @pocmo, do you remember what `plural_rule` is in this structure? I was talking with Delphine and she wasn't sure either.

```
'name': 'Occitan',
'local_name': 'occitan',
'plural_rule': 'fi',
'team': 'oc <LL@li.org>\n'
```

**Comment by pocmo** [comment]
Yes!

Those are locales where Android2Po (a2po) has no information about. Here a2po needs to know how plurals are handled in those locales. So in our workaround we tell it to apply the rules from another locale it already knows about.

To know which locale to pick I looked at the po files in the l10n repository:
https://github.com/mozilla-l10n/focus-android-l10n/tree/master/locales

So for `ixl`:
https://github.com/mozilla-l10n/focus-android-l10n/blob/a2efea08aac9ac9137966bf84cc3505cc00637d0/locales/ixl/app.po#L19

```
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

I then picked an other (more common) locale with the same rule.

For `ixl` this could be `de`:
https://github.com/mozilla-l10n/focus-android-l10n/blob/a2efea08aac9ac9137966bf84cc3505cc00637d0/locales/de/app.po#L19

```
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```
